getting_started/build-guide/build-from-scratch.rst

```diff
@@ -12,21 +12,25 @@ Intel platforms include: |BYT|, |CHT|, |HSW|, |BDW|, |APL|, and |CNL|.
 
 Support also exists for NXP i.MX8/i.MX8X/i.MX8M platforms.
 
-Build SOF binaries
-******************
+Build SOF
+*********
+
 The following steps describe how to install the SOF development environment
-on Ubuntu 16.04, 18.04, and 18.10.
+on Ubuntu 16.04, 18.04, and 18.10. They should work on other Linux
+distributions with minor modifications.
```

Comment:
We could also add these should also work too on Ubuntu 19.04 & 19.10 with possible minor changes. I think most devs are on 19.10.